File: Daily Test/Python/Even Number Of Digits 4.4.2026.py
arr = list(map(int, input().split()))

for num in arr:
    if len(str(num)) % 2 == 0:
        print(num, end=" ")


//Without Built-in 

int digits = 0;
int temp = num;

while(temp > 0){
    digits++;
    temp /= 10;
}

File: Daily Test/C++/Even Number Of Digits 4.4.2026.cpp
#include <iostream>
using namespace std;

int main() {
    int n;
    cin >> n;

    for(int i = 0; i < n; i++) {
        int num;
        cin >> num;

        int digits = 0;
        int temp = num;

        if(temp == 0) digits = 1;

        while(temp > 0) {
            digits++;
            temp /= 10;
        }

        if(digits % 2 == 0) {
            cout << num << " ";
        }
    }

    return 0;
}